source/components/Dropdown/index.js

```diff
@@ -104,42 +133,77 @@ Dropdown.propTypes = {
      * React Component to display as the Dropdown Content
      */
     children: PropTypes.node,
+
     /**
-     * Whether or not dropdown should have a slight drop shadow
+     * Should dropdown content be scrollable
      */
     contentScrollable: PropTypes.bool,
+
     /**
-     * Hides chevron in dropdown toggle
+     * Should dropdown content be left-aligned with the dropdown toggle
      */
     dropdownLeftAligned: PropTypes.bool,
+
     /**
-     * Whether or not dropdown should have a drop shadow (darker than the one produced by hasShadow)
+     * Should dropdown content be right-aligned with the dropdown toggle
      */
     dropdownRightAligned: PropTypes.bool,
+
     /**
-     * Is it a nested dropdown
+     * Whether or not dropdown should have a drop shadow (darker than the one produced by hasShadow)
      */
     hasDarkShadow: PropTypes.bool,
+
     /**
-     * Should dropdown content be left-aligned with the dropdown toggle
+     * Whether or not dropdown should have a slight drop shadow
      */
     hasShadow: PropTypes.bool,
+
     /**
      * is active
      */
     isActive: PropTypes.bool,
+
     /**
-     * Should dropdown content be right-aligned with the dropdown toggle
+     * Is it a nested dropdown
      */
     isLevel2: PropTypes.bool,
+
     /**
-     * Should dropdown content be scrollable
+     * if the top of nested dropdown content should be positioned at the same height as toggle
+     */
+    isStickedToParent: PropTypes.bool,
+
+    /**
+     * Hides chevron in dropdown toggle
      */
     noChevron: PropTypes.bool,
+
+    /**
+     * Removes span around element passed in the "toggle" prop
+     */
+    shouldNotWrapToggle: PropTypes.bool,
```

Comment:
What's the use case for this property? - it's missing in the examples.
Looks like whenever we should wrap or not should be decided based on the number of elements or the fact if they're text nodes or not so the component can decide if it's supposed to wrap it or not.
Right now this prop decides of the implementation which can change - it's better if the props describe the properties/behaviors of the component and not reference the internal logic of the component.

Reply:
Removed, we will not have a case for it anymore.